docs/guides/primitives.mdx

```diff
@@ -27,6 +27,10 @@ In short, the computational model introduced by the Qiskit primitives moves quan
 to where classical programming is today, where the focus is less on the hardware details and more on the results
 you are trying to achieve.
 
+<Admonition type="note">
+The V1 primitives have been deprecated.  For instructions to migrate to the V2 primitives, see the [migration guide.](/api/migration-guides/v2-primitives)
```

Comment:
So, we only deprecated Qiskit Runtime's implementation of V1 primitive. The interface in Qiskit SDK is _not_ deprecated (just to make things more confusing 😞)